data/projects/zkevm-community.ts

```diff
@@ -15,8 +15,9 @@ export const zkevmCommunity: ProjectInterface = {
     github: "https://github.com/privacy-scaling-explorations/zkevm-circuits",
   },
   tags: {
-    themes: ["Scaling"],
+    keywords: ["Scaling"],
+    themes: ["build"],
     types: ["Infrastructure/protocol", "Lego sets/toolkits"],
-    builtWith: ["Halo2 from zcash", "Rust", "geth"],
+    builtWith: ["halo2 from zcash", "rust", "geth"],
```

Comment:
need to check on these. if we can keep these consistent we should. (not sure why its" halo2 from x") instead of just halo2

Reply:
halo2